easybuild/easyconfigs/f/fmt/fmt-8.1.1-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'CMakeMake'
+
+name = 'fmt'
+version = '8.1.1'
+
+homepage = 'http://fmtlib.net/'
+description = "fmt (formerly cppformat) is an open-source formatting library."
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/fmtlib/fmt/releases/download/%(version)s/']
+sources = ['fmt-%(version)s.zip']
+checksums = ['23778bad8edba12d76e4075da06db591f3b0e3c6c04928ced4a7282ca3400e5d']
+
+builddependencies = [
+    ('CMake', '3.22.1'),
+    ('binutils', '2.37')
+]
+
+separate_build_dir = True
+
```

Comment:
```suggestion

# build static and shared libraries
configopts = ["-DCMAKE_POSTITION_INDEPENDENT_CODE=TRUE", "-DBUILD_SHARED_LIBS=TRUE"]

runtest = 'test'

```

Reply:
might as well build the shared library as well, and run the tests

Reply:
I agree. However,  thought this is what:
`toolchainopts = {'pic': True}`
does. Is that not the case?

Reply:
good point, you could also leave the toolchainopts and use `configopts = ["", "-DBUILD_SHARED_LIBS=TRUE"]`

Reply:
Is there a preferred way of doing this or is that just a matter of taste?

Reply:
either should be fine